src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -3021,7 +3028,8 @@ def set_up_network_profile(self, mc: ManagedCluster) -> ManagedCluster:
         if advanced_network_observability is not None:
             network_profile.advanced_networking = self.models.AdvancedNetworking(  # pylint: disable=no-member
```

Comment:
See comment about AdvancedNetworking model in #7860

Reply:
Thank you for catching that updated!